app/informativedialog.h

```diff
@@ -0,0 +1,23 @@
+#ifndef INFORMATIVEDIALOG_H
+#define INFORMATIVEDIALOG_H
+
+#include <QDialog>
+
+namespace Ui
+{
+class InformativeDialog;
+}
+
+class InformativeDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit InformativeDialog(QString title, QString description = QString(), QWidget *parent = 0 );
+    ~InformativeDialog();
+
+private:
+    Ui::InformativeDialog *ui;
+};
```

Comment:
Is there any new feature will be added into InformativeDialog? QMessagebox::information() is doing the same job.